server/catgenome/build.gradle

```diff
@@ -163,6 +163,7 @@ dependencies {
     testCompile("org.springframework.boot:spring-boot-starter-test")
     testCompile group: "org.unitils", name: "unitils-core", version: project.ext.versionUnitils
     testCompile group: "javax.servlet", name: "javax.servlet-api", version: project.ext.versionJavaXServlet
+    testCompile group: "javax.transaction", name: "jta", version: "1.1"
```

Comment:
I don't see, why you need this in tests.

Reply:
Changed group to testRuntime, because this library is needed for EhCache tests.